SpinCore_pp/run_IR_STE.py

```diff
@@ -169,7 +169,7 @@ def verifyParams():
         SpinCore_pp.stop_ppg();
         print("\nRUNNING BOARD...\n")
         SpinCore_pp.runBoard();
-        raw_data = SpinCore_pp.getData(data_length, nPoints, nEchoes, nPhaseSteps, output_name)
+        raw_data = SpinCore_pp.getData(data_length, nPoints, nEchoes, nPhaseSteps)
         raw_data.astype(float)
         data_array = []
         data_array[::] = np.complex128(raw_data[0::2]+1j*raw_data[1::2])
```

Comment:
talked with alec and can probs be removed. this was to produce a stimulated echo for farhana's oil flow experiments